docs/how-to/document-plugin.rst

```diff
@@ -31,6 +27,9 @@ necessary:
 It is also useful to indicate if the plugin replaces an existing plugin, or
 should be used instead of another similar plugin.
 
+If documenting an application-specific plugin, it may be appropriate to include
```

Comment:
How about:

```suggestion
If you're documenting an application-specific plugin, we encourage you to include
```

Reply:
It's not necessarily encouraged though.

Reply:
Oh, I thought this meant that non-application-specific plugins should never have versions, but some application-specific ones should. What are we advising for other plugins?

Reply:
For generic craft-parts plugins, they should not have a version. For application-specific ones, it depends on the app. If the app uses versioned documentation, it's not necessary. If the app's documentation is unversioned, it might make sense.

Reply:
What do you think about including all that nuance? I'm not sure I could grok that from this template.

Reply:
That seems more like something that's up to the app development team.

Reply:
I've removed the paragraph as a whole.